dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -633,3 +633,19 @@ ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host n
 ssl.client.handshake.error.CWPKI0824E.explanation=An error occurred during the SSL handshake.  Host name verification is enabled and server's identity can not be verified.  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.
 ssl.client.handshake.error.CWPKI0824E.useraction=When host name verification is enable the server's being accessed needs to be setup with a certificate contain the proper information needed to verify the server's identity.
 # -------------------------------------------------------------------------------------------------
+ssl.cacerts.error.CWPKI0825W=CWPKI0825W: The trustDefaultCerts attribute is enabled and the default truststore failed to initialize.  Only the configured truststore will be used, and the default truststore will not.  The [{0}] SSL configuration will be used without the default truststore.  The exception returned is: [{1}].
+ssl.cacerts.error.CWPKI0825W.explanation=A problem occurred while trying to setup the default truststore.  It will not be used for trust.  Only the configured SSL configuration will be used.
+ssl.cacerts.error.CWPKI0825W.useraction=Ensure the default truststore file exits and is accessible.
+# -------------------------------------------------------------------------------------------------
+ssl.environment.cert.error.CWPKI0826W=CWPKI0826W: An error occurred while trying to obtain the certificate from the [{0}] environment variable.  The certificate will not be added to the [{1}] truststore.
```

Comment:
Change
"The certificate **will not be** added to the [{1}] truststore."
To
"The certificate **is** not added to the [{1}] truststore."